ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss

```diff
@@ -0,0 +1,391 @@
+        }
+
+        .title {
+            margin-top: 40px;
+            margin-bottom: 34px;
+        }
+
+        .text {
+            margin-bottom: 25px;
+        }
+
+        .join-cmi-btn {
+            margin: 0;
+        }
+    }
+}
+
+.title, .text {
+    max-width: 728px;
+    margin: 0 auto;
+    text-align: center;
+}
+
+.title {
+    color: #000000;
+    font-family: Source Serif Pro;
+    font-size: 40px;
+    font-weight: 600;
+    letter-spacing: -0.08px;
+    line-height: 50px;
+}
+
+.text {
+    opacity: 0.8;
+    color: #000000;
+    font-family: Untitled-Sans-Regular;
+    font-size: 24px;
+    letter-spacing: -0.1px;
+    line-height: 34px;
+}
+
+.btn {
+    font-family: Untitled-Sans-Medium;
+    font-size: 18px;
+    letter-spacing: -0.26px;
+    padding: 15px 30px;
+    color: #000000;
+    line-height: 22px;
+    text-align: center;
+    border: 2px solid #000000;
+    border-radius: 4px;
+    text-decoration: none;
+
+    &:focus {
+        outline: none;
+        box-shadow: 0 0 0 4px rgba(21, 156, 228, 0.4);
+    }
+}
+
+.join-cmi-btn {
+    background-color: #7154FF;
+    color: #ffffff;
+    font-size: 20px;
+    letter-spacing: 0;
+    margin: 25px 0 120px;
+    border: none;
+}
+
+.learn-more-btn {
+    margin-top: 45px;
+}
+
+.center {
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+}
+
+@media screen and (max-width: $desktop-breakpoint) {
+    section {
+        &:nth-child(n) {
+            padding-left: 5rem;
+            padding-right: 5rem;
+            height: auto;
+        }
+
+        &.introduction {
+            background-size: 90%;
+        }
+    }
+}
+
+@media screen and (max-width: $tablet-breakpoint) {
+    section {
+        &:nth-child(n) {
+            padding-left: 3rem;
+            padding-right: 3rem;
+        }
+
+        &.join {
+            flex-direction: column;
+
+            .image {
+                margin-right: 0;
+            }
+        }
+
+        &.participate {
+            .steps {
+                flex-wrap: wrap;
+                gap: 3rem;
+            }
+
+            .step {
+                flex: 1 1 200px;
+                margin-right: 0;
+
+                &__image {
+                    width: 250px;
+                    height: 250px;
+                    left: unset;
+                }
+
+                &__description {
+                    max-width: unset;
+                }
+            }
+        }
+
+        &.faq {
+            .container {
+                padding: 0;
+            }
+        }
+
+        &.stay-informed {
+            .social-media {
+                flex-direction: column;
+                gap: 3rem;
+                align-items: start;
+
+                &__links {
+                    flex-basis: 50px;
+                    gap: 3rem;
+                }
+            }
+        }
+
+        .join-cmi-logo {
+            width: 95%;
+        }
+
+        .btn {
+            font-size: 1.4rem;
```

Comment:
I think the idea is that we need to set the size in em/rem for text and use @media once to set different font-size for body in order to calculate rem/em correctly

Reply:
Added at 8e4e176298eb4955d2838fd1abe0c06da8731fdb